plugins/helper/database/credsutil/credsutil.go

```diff
@@ -16,24 +18,35 @@ type CredentialsProducer interface {
 	GenerateExpiration(ttl time.Time) (string, error)
 }
 
-// RandomAlphaNumericOfLen returns a random byte slice of characters [A-Za-z0-9]
+const (
+	reqStr    = `A1a`
+	minStrLen = 10
+)
+
+// RandomAlphaNumeric returns a random string of characters [A-Za-z0-9]
 // of the provided length.
-func RandomAlphaNumericOfLen(len int) ([]byte, error) {
-	retBytes := make([]byte, len)
-	size := 0
+func RandomAlphaNumeric(length int) (string, error) {
```

Comment:
Could you add a comment specifying the prepended string takes up 3 characters of space?